File: sample/src/main/java/contacts/sample/cheatsheet/sim/java/InsertSimContactsActivity.java
package contacts.sample.cheatsheet.sim.java;

import android.app.Activity;

import contacts.core.ContactsFactory;
import contacts.core.entities.NewSimContact;
import contacts.core.sim.SimContactsInsert;

public class InsertSimContactsActivity extends Activity {

    SimContactsInsert.Result insertSimContact() {
        return ContactsFactory.create(this)
                .sim()
                .insert()
                .simContacts(new NewSimContact("Mr. Joe", "5555555555"))
                .commit();
    }
}

File: sample/src/main/java/contacts/sample/cheatsheet/sim/kotlin/InsertSimContactsActivity.kt
package contacts.sample.cheatsheet.sim.kotlin

import android.app.Activity
import contacts.core.Contacts
import contacts.core.sim.SimContactsInsert

class InsertSimContactsActivity : Activity() {

    fun insertSimContact(): SimContactsInsert.Result = Contacts(this)
        .sim()
        .insert()
        .simContact {
            name = "Mr. Joe"
            number = "5555555555"
        }
        .commit()
}